src/utils/yelpFusionApiUtils.ts

```diff
@@ -0,0 +1,37 @@
+import yelp from 'yelp-fusion';
+import { YelpClient } from 'yelp-fusion';
+import { IConfiguration } from '../models/data_models/IConfiguration.js';
+import { Geocode } from '../models/data_models/types.js';
+
+export class YelpFusionApiUtils {
+	private yelpClient?: YelpClient;
+
+	constructor(private config: IConfiguration) {}
+
+	public async init(): Promise<YelpFusionApiUtils> {
+		const apiKey = this.config.yelp.apiKey;
+		this.yelpClient = yelp.client(apiKey);
+		return this;
+	}
+
+	public async getImages(name: string, geocode: Geocode): Promise<string[]> {
```

Comment:
ooh I've been using `Array<string>` over `string[]`

they're the same thing but maybe we should decide on a convention